ch_integration_tests/tests/integration.rs

```diff
@@ -66,17 +67,23 @@ mod tests {
             kernel_path.push("workloads");
 
             match self {
-                KernelType::RustFw => {
+                KernelType::Direct => {
                     #[cfg(target_arch = "aarch64")]
                     kernel_path.push("Image");
                     #[cfg(target_arch = "x86_64")]
-                    kernel_path.push("hypervisor-fw");
+                    kernel_path.push("vmlinux");
                 }
-                KernelType::Direct => {
+                KernelType::OvmfFw => {
                     #[cfg(target_arch = "aarch64")]
                     kernel_path.push("Image");
                     #[cfg(target_arch = "x86_64")]
-                    kernel_path.push("vmlinux");
+                    kernel_path.push("OVMF-4b47d0c6c8.fd");
+                }
+                KernelType::RustFw => {
+                    #[cfg(target_arch = "aarch64")]
```

Comment:
What've you done is fine. But I think we should `assert!` if you try and combine aarch64 with !Direct.

Reply:
Sounds good!

Reply:
@rbradford fixed! PTAL